exporter/exporter.go

```diff
@@ -267,6 +268,21 @@ func NewRedisExporter(redisURI string, opts Options) (*Exporter, error) {
 			"server_threads":        "server_threads_total",
 			"long_lock_waits":       "long_lock_waits_total",
 			"current_client_thread": "current_client_thread",
+
+			// Redis Modules metrics
+			// RediSearch module
+			"search_number_of_indexes":   "search_number_of_indexes",
+			"search_used_memory_indexes": "search_used_memory_indexes_bytes",
+			"search_total_indexing_time": "search_total_indexing_time_ms",
+			"search_global_idle":         "search_global_idle",
+			"search_global_total":        "search_global_total",
+			"search_bytes_collected":     "search_collected_bytes",
+			"search_total_cycles":        "search_total_cycles",
+			"search_total_ms_run":        "search_total_run_ms",
+			"search_dialect_1":           "search_dialect_1",
+			"search_dialect_2":           "search_dialect_2",
+			"search_dialect_3":           "search_dialect_3",
+			"search_dialect_4":           "search_dialect_4",
```

Comment:
One more thing - I haven't added metrics collection configuration section because it's kinda static. It could be changed via module settings, but they're not changing by itself.

I'll revisit this later when have more statistics on this module usage.

```
# search_runtime_configurations
search_enableGC:ON
search_minimal_term_prefix:2
search_minimal_stem_length:4
search_maximal_prefix_expansions:200
search_query_timeout_ms:500
search_timeout_policy:return
search_cursor_read_size:1000
search_cursor_max_idle_time:300000
search_max_doc_table_size:1000000
search_max_search_results:1000000
search_max_aggregate_results:-1
search_gc_scan_size:100
search_min_phonetic_term_length:3
```

Reply:
Yeah, we can safe that for later.